src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java

```diff
@@ -654,6 +656,77 @@ public JsonObject getBingoInformation() {
 			return bingoInformation != null ? bingoInformation : null;
 		}
 
+		/**
+		 * Returns -1 if error
+		 * Returns -2 if still loading
+		 */
+		public long getSoopyNetworth(String profileName){
+			if (profileName == null) profileName = latestProfile;
+			if (soopyNetworth.get(profileName) != null) return soopyNetworth.get(profileName);
+
+			JsonArray playerInfo = getSkyblockProfiles(() -> {});
+			if (playerInfo == null) return -2;
+			if (updatingSoopyNetworth.get()) return -2;
+			updatingSoopyNetworth.set(true);
+
+			manager.hypixelApi.postApiAsync(
+				"http://soopy.dev/api/v2/player_networth/"+this.uuid, //TODO: change to https when lets encript certificate is supported
```

Comment:
Working on it

Reply:
I had it as https and it was working, but idk if thats something with my pc, or lets encript being supported in the api requests

Reply:
Currently the only the pronoundb uses a custom keystore, normal requests use the JVM keystore. Since the standard minecraft launcher uses 1.8u51 lets encrypt isnt supported in those requests yet.